transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -475,7 +462,7 @@ public void selected(final MappingOperation<?, ?> mapping) {
      */
     @Override
     public void setFocus() {
-    	
+
     }
```

Comment:
Oh, never mind, it is required to implement; there just isn't anything really appropriate to give focus when the editor is first displayed.